tasks/fix-a-bug.md

```diff
@@ -0,0 +1,32 @@
+#Labyrinth: Fix a bug
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+We can add new features to the labyrinth which improve the game.
+
+Requirements:
+- Create or work on an  existing issue(Please take permission from the opener) to solve a bug in the site.
+  These are different aspects which you can cover:
```

Comment:
add: "If you create an issue: These are different aspects which you should cover:"